pinkie-card/GenericTileIcon.qml

```diff
@@ -0,0 +1,135 @@
+import QtQuick 2.7
```

Comment:
There seems to be some duplicate code between this file and the other tile type one.